lab-taylor/app/html/index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html ng-app="adventureGame">
+  <head>
+    <meta charset="utf-8">
+    <title>__adventure-game__</title>
+    <link rel="stylesheet" href="bundle.css">
+    <script type="text/javascript" src="bundle.js"></script>
+  </head>
+  <body ng-controller="GameController as gameCtrl" ng-class="gameCtrl.dynamicClass">
+    <h1>Lost In Space</h1>
+    <main >
+      <form novalidate name="gameForm">
+        <label>Select a Direction</label>
+        <select ng-model="gameCtrl.gameForm.direction">
+          <option value="forward"> Forward </option>
+          <option value="backward"> Backward </option>
+          <option value="left"> left </option>
+          <option value="right"> right </option>
+        </select>
+
+        <input type="button" value="MOVE" ng-click="gameCtrl.moveDirection(gameCtrl.gameForm.direction)">
```

Comment:
You have created a form here, and remember that angular actually overrides the html and has its own form directive, so form with name "gameForm" has its own scope. This means you can get rid of `gameCtrl.` in the form and just use `gameForm.direction`. To set your initial direction just use ng-init `<select ng-init="gameForm.direction = 'forward'" ...other code>`